src/ts/uimanager.ts

```diff
@@ -1001,6 +1001,97 @@ export namespace UIManager.Factory {
   }
 }
 
+// Separate the demos from the other factories.
+// The others are more like different 'styles' than some variants with different features
+export namespace UIManager.Demo.Factory {
+  export function buildDemoWithSeparateAudioSubtitlesButtons(player: PlayerAPI, config: UIConfig = {}): UIManager {
+    // show smallScreen UI only on mobile/handheld devices
+    let smallScreenSwitchWidth = 600;
+
+    return new UIManager(player, [{
+      ui: UIManager.Factory.modernSmallScreenAdsUI(),
+      condition: (context: UIConditionContext) => {
+        return context.isMobile && context.documentWidth < smallScreenSwitchWidth && context.isAdWithUI;
+      },
+    }, {
+      ui: UIManager.Factory.modernAdsUI(),
+      condition: (context: UIConditionContext) => {
+        return context.isAdWithUI;
+      },
+    }, {
+      ui: UIManager.Factory.modernSmallScreenUI(),
+      condition: (context: UIConditionContext) => {
+        return context.isMobile && context.documentWidth < smallScreenSwitchWidth;
+      },
+    }, {
+      ui: modernUIWithSeparateAudioSubtitlesButtons(),
+    }], config);
+  }
+
+  function modernUIWithSeparateAudioSubtitlesButtons() {
+    let subtitleOverlay = new SubtitleOverlay();
+
+    let settingsPanel = new SettingsPanel({
+      components: [
+        new SettingsPanelItem('Video Quality', new VideoQualitySelectBox()),
+        new SettingsPanelItem('Speed', new PlaybackSpeedSelectBox()),
+        new SettingsPanelItem('Audio Quality', new AudioQualitySelectBox()),
+      ],
+      hidden: true,
+    });
+
+    let controlBar = new ControlBar({
+      components: [
+        settingsPanel,
+        new Container({
+          components: [
+            new PlaybackTimeLabel({ timeLabelMode: PlaybackTimeLabelMode.CurrentTime, hideInLivePlayback: true }),
+            new SeekBar({ label: new SeekBarLabel() }),
+            new PlaybackTimeLabel({ timeLabelMode: PlaybackTimeLabelMode.TotalTime, cssClasses: ['text-right'] }),
+          ],
+          cssClasses: ['controlbar-top'],
+        }),
+        new Container({
+          components: [
+            new PlaybackToggleButton(),
+            new VolumeToggleButton(),
+            new VolumeSlider(),
+            new Spacer(),
+            new PictureInPictureToggleButton(),
+            new AirPlayToggleButton(),
+            new CastToggleButton(),
+            new VRToggleButton(),
+            new AudioTrackSelectBox({
```

Comment:
All settings must be within a `SettingsPanel`. It doesn't look nice to display a `SelectBox` out of nowhere without a container.

Reply:
See #121